SOAR-2883 Update help and add filter to only allow assets scanned in last 30 days (#402)
rapid7/insightconnect-workflows: workflows/InsightVM_Top_25_Remediations_to_JIRA/help.md

# Description

> This workflow checks assets scanned in the last 30 days. For each of the 25 assets with the highest vulnerability score, it creates one solution-based issue in JIRA per vulnerability found. These tickets list the asset(s) affected. If there is already an issue open for a given remediation, the workflow updates the existing issue with a current list of impacted assets.

Reviewer: ```suggestion
This workflow checks assets scanned in the last 30 days. For each of the 25 assets with the highest vulnerability score, it creates one solution-based issue in Jira per vulnerability found. These tickets list the asset(s) affected. If there is already an issue open for a given remediation, the workflow updates the existing issue with a current list of impacted assets.
```

* In the "Process Remediations" loop, change the "Search for Existing Issue" step's JQL input. "change_me" must be the name of the project in JIRA that will be searched for issues. 
* In the "Process Remediations" loop, change the "Create Issue" step's Project input. "change_me" must be the ID of the project in JIRA where the issue will be created.

> This workflow runs daily at 12:00 UTC. Adjust the time as necessary to suit your organization's needs by changing input in "Time in UTC." We recommend running this workflow during off-hours, as it is data and CPU intensive.

Reviewer: ```suggestion
This workflow runs daily at 12:00 UTC. Adjust the time as necessary to suit your organization's needs by changing input in "Time in UTC." It is recommended to run this workflow during off-hours, as it is data and CPU-intensive.
```

> Note: Editing any of the following can significantly increase or decrease the time this workflow takes to run.

Reviewer: ```suggestion
Note: Editing any of the following may negatively impact the runtime duration of the workflow.
```

> By default, this workflow gets assets scanned in the last 30 days. To alter the limit from 30 days to a different number, edit the `Get assets scanned in the last 30 days` step. Find the input variable {"match":"all","filters":[{"field":"last-scan-date","operator":"is-within-the-last","value":30}]}. Change the number 30 within this variable to fit the needs of your organization.

Reviewer: ```suggestion
By default, this workflow gets assets scanned in the last 30 days. To alter the limit from 30 days to a different number, edit the `Get Assets Scanned in the Last 30 Days` step. Find the input variable {"match":"all","filters":[{"field":"last-scan-date","operator":"is-within-the-last","value":30}]}. Change the number 30 within this variable to fit the needs of your organization.
```

> To change the limit for number of assets scanned, go to the `Top 25 Remediations` step. Edit the `Asset Limit` field. To change the limit number of vulnerabilities found, you can edit the `Vulnerability limit` Field

Reviewer: ```suggestion
To change the limit for number of assets scanned, go to the `Top 25 Remediations` step. Edit the `Asset Limit` field. The `Vulnerability Limit` field can be adjusted to limit the number of vulnerabilities.
```
